store/index.js

```diff
@@ -51,12 +59,35 @@ export const getters = {
 }
 
 export const mutations = {
+  setCurrentDB(state, db) {
+    state.currentDB = db
+  },
+  setFileManager(state, fileManager) {
+    state.fileManager = fileManager
+  },
   setSelectedFiles(state, { key, value }) {
     state.selectedFiles[key] = value
   },
   setResult(state, { company, experience, result }) {
     state.results[company][experience] = result
   },
+  clearStore(state) {
+    state.selectedFiles = Object.fromEntries(
+      config.experiences.map(k => [k, []])
+    )
+    state.results = Object.fromEntries(
+      config.experiences.map(k => [
+        k,
+        Object.fromEntries(
+          manifestMap[k].defaultView?.map(b => [b.key, null]) ?? []
+        )
+      ])
+    )
+    state.fileExplorerCurrentItem = {}
+    if (state.currentDB !== null) state.currentDB.close()
+    state.currentDB = null
+    state.fileManager = null
```

Comment:
This function could call the other mutations, I would find it cleaner:
```js
for (const company of config.experiences) {
  this.commit('setSelectedFiles', { key: company, value: [] })
  for (const experience of manifestMap[company]) {
    this.commit('setResult', { company, experience, value: null }
  }
}
this.commit('setCurrentDB', null)
this.commit('setFileManager', null)
```

This could also be turned into an action.

Also, note that currently it clears the results for **all** the experiences. I'm not sure if we want this

Reply:
We already had one FileManager and one database object shared by all experiments? So there is no need to keep the states of the other experiments, right?

Reply:
Ah yes you're right. So we should decide on the direction we wanna take, either:

1. Only keep the state of one experiment. In this case we can simplify `results` and `selectedFiles`.
1. Keep the state of the other experiments. In this case we have to store multiple `fileManager` and `db`

Let's discuss this in the meeting

Reply:
As discussed in the meeting we don't need to keep the state of every experiment, and doing so would complicate the code so let's simplify `results` and `selectedFiles`